api/src/main/java/io/kafbat/ui/serdes/builtin/ProtobufFileSerde.java

```diff
@@ -411,7 +411,10 @@ private Map<String, ProtoFile> loadFilesWithLocations() {
               // relative path will be used as "import" statement
               String relativePath = baseLocation.relativize(path).toString();
               var protoFileElement = ProtoParser.Companion.parse(
-                  Location.get(baseLocation.toString(), relativePath),
+                  Location.get(
```

Comment:
Hi, thanks for the PR.
This is what I meant in https://github.com/kafbat/kafka-ui/pull/696#issuecomment-2528663163. This feels like a hacky solution aiming to fix the tests on Windows. In the end, there are built-in APIs in io/nio that we'd like to use if there's a need.
it's been discussed here before: https://github.com/kafbat/kafka-ui/pull/261#discussion_r1606089719
Considering this is a bug in wire, I'd wait for it to get fixed so we don't have to resort to such solutions.

Let me know what you think

Reply:
Sure, we can wait for some fix from 'wire' side and I understnand that it looks hacky, that is why I higlighted this approach as tempo in comments. Feel free to close it.